vpc.tf

```diff
@@ -0,0 +1,109 @@
+# VPC
+resource "aws_vpc" "task1_vpc" {
+  cidr_block       = var.vpc_cidr
+  instance_tenancy = "default"
+  tags = {
+    Name = "task1_vpc"
+  }
+}
+
+# Internet Gateway For Public Setup
+resource "aws_internet_gateway" "task1_igw" {
+  vpc_id = aws_vpc.task1_vpc.id
+  tags = {
+    Name = "task1_igw"
```

Comment:
Please introduce the `environemnt_code` variable and interpolate it in each Name tag